src/icons/Search.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Svg, Path } from "react-native-svg";
+
+type Props = {size: number | string,
+              color: string }
+export const Search = ({size, color}: Props) => {
```

Comment:
Let's give this some defaults. Size 24 (may need to demo to Ethan, use ur judgement), Color (confer with Ethan)